src/main/java/carRacing/model/RacingCarName.java

```diff
@@ -0,0 +1,25 @@
+package carRacing.model;
+
+public class RacingCarName {
+    private String name;
+    private static final int MAX_LENGTH = 5;
```

Comment:
👍
자바 컨벤션을 따르면 상수는 멤버변수보다 위에 있는 것이 맞습니다.

https://www.oracle.com/java/technologies/javase/codeconventions-fileorganization.html
3.1.3 부분을 읽어보시면 도움이 될거에요

```suggestion
public class RacingCarName {
    private static final int MAX_LENGTH = 5;
    
    private String name;
```

Reply:
이름이 변경된다는 요구사항은 없으므로 final을 붙여 불변을 유지하면 좋겠네요